### DISCUSSION TITLE
Two large diffdisk files

### DISCUSSION THREAD

**Discussion by RyanRoberts** [abiosoft/colima#920] [Q&A]
Hey folks. Is it right that I have two large diffdisk files used by Colima? I have the following:

- ~/.colima/_lima/colima/diffdisk - 14.8GB
- ~/.lima/colima/diffdisk - 29.5GB

The second one looks about right to me but I'm unsure about that first one, I've never seen two file on my system the past. 

The directory contents and structure of the `~/.lima/colima` and `~/.colima/_lima/colima` are near identical.

I'm using DDEV so I'm no expert in Lima or Colima.

Thanks

**Comment by RyanRoberts** [comment]
Sorted it. I see the 0.6.0 update moved the contents to that second directory.

**Reply by alexanderbaumann-toast** [reply]
Can we delete `~/.colima/_lima/colima/diffdisk` then?

**Reply by fessyfoo** [reply]
IIUC that's the new directory.
